src/datagen/main/java/mekanism/client/lang/MekanismLangProvider.java

```diff
@@ -1296,6 +1297,7 @@ private void addMisc() {
         add(MekanismModules.GRAVITATIONAL_MODULATING_UNIT, "Gravitational Modulating Unit", "Using experimental technologies and the tremendous energy of antimatter, allows the user to defy gravity.");
         add(MekanismModules.ELYTRA_UNIT, "Elytra Unit", "Applies an HDPE Reinforced Elytra to the MekaSuit.");
         add(MekanismModules.GYROSCOPIC_STABILIZATION_UNIT, "Gyroscopic Stabilization Unit", "Allows the user to act as though they are on solid ground.");
+        add(MekanismModules.HYDROSTATIC_REPULSOR_UNIT, "Hydrostatic Repulsor Unit", "Allows the user to move through water with less resistance.");
```

Comment:
Thoughts on if this might be better as a description?
> Uses advanced technology to repel water lowering the resistance felt while moving through it.